File: Volume 11 (1100 - 1199)/1134/1134.py
#!/usr/bin/env python3

l = [0,0,0]
n = int(input())
while n != 4:
    if n == 1: l[0] += 1
    elif n == 2: l[1] += 1
    elif n == 3: l[2] += 1
    n = int(input())
print("MUITO OBRIGADO\nAlcool: {}\nGasolina: {}\nDiesel: {}".format(l[0],l[1],l[2]))

File: Volume 11 (1100 - 1199)/1134/1134.c
#include <stdio.h>
 
int main() {
    int n, a = 0, g = 0, d = 0;
    while (scanf("%d", &n) && n != 4) {
        if (n == 1) a++;
        else if (n == 2) g++;
        else if (n == 3) d++;
    }
    printf("MUITO OBRIGADO\nAlcool: %d\nGasolina: %d\nDiesel: %d\n", a, g, d);
    return 0;
}